DrewBradley/rancid-tomatillos
# Feature/accessibility and responsiveness

### What does this PR do?
- This PR adds media breakpoints and styling to make the app responsive. It also adds aria labels and alt text to achieve an accessibility score of 100 for both our main display and individual views

### What does it fix?
- No bugs to fix.

### Is this a feature or a fix?
- This is a feature, adding responsiveness and accessibility.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start. 


### How should this be tested?
- Open dev tools on the site. Check different mobile views and try resizing the screen to check responsiveness. Run lighthouse audit on main display, then click on a movie and run another lighthouse audit on the individual movie view to check accessibility score is 100.
